src/views/teachers/landing/landing.jsx

```diff
@@ -22,107 +22,255 @@ const Landing = () => (
                     <p className="title-banner-p intro">
                         <FormattedMessage id="teacherlanding.intro" />
                     </p>
-                    <div className="ted-talk">
+                    <div className="video-player">
                         <iframe
                             allowFullScreen
+                            allowTransparency="true"
                             frameBorder="0"
-                            src="https://www.youtube.com/embed/uPSuG063jhA?border=0&wmode=transparent"
+                            height="180"
+                            scrolling="no"
+                            src={`https://fast.wistia.net/embed/iframe/h8hay3nnt4?seo=false&videoFoam=true`}
+                            title=""
+                            width="320"
                         />
                     </div>
                 </FlexRow>
             </div>
             <div className="band">
                 <SubNavigation className="inner">
-                    <a href="#in-practice">
+                    <a href="#resources">
                         <li>
-                            <FormattedMessage id="teacherlanding.inPracticeTitle" />
+                            <FormattedMessage id="teacherlanding.resourcesAnchor" />
```

Comment:
I wouldn't use "Anchor" in the name -- it happens to be at the same place as the anchor, but it's really a title, or a section title. I think something with "title" rather than "anchor" would be more clear.